grr/server/grr_response_server/gui/ui/components/expandable_hash/expandable_hash.ts

```diff
@@ -1,22 +1,35 @@
 import { Component, OnInit, Input } from '@angular/core';
 import { Hash } from '../../lib/api/api_interfaces'
 
-export const MAX_CHARACTERS_IN_TRUNCATED_HASH = 9; // TODO: Check
-export const SHA_256_NA_MESSAGE = 'sha256 n/a' // TODO: Check
+/** Max number of hash characters to be displayed when not hovered */
+export const TRUNCATED_HASH_CHAR_LIMIT = 7; // TODO: Check
+
+/** Message to display if SHA-256 is undefined */
+export const SHA_256_NA_MESSAGE = 'SHA-256 n/a' // TODO: Check
+
+/** The 'Horizontal ellipsis' character: '…' */
+export const ELLIPSIS = '\u2026';
```

Comment:
You can use the UTF-8 character `…` directly in strings for better readability, no need for the escape sequence  `\u2026`.